sandbox/scratch_gen_functions.R

```diff
@@ -0,0 +1,361 @@
+#' Calculate USHE Element Institution (Institution)
```

Comment:
I am absolutely BLOWN away by the quality of code here! 

We should definitely get these general functions integrated with the base package. This will eliminate maintenance of many lines of code. 

Seriously, A+ code here. Thank you! @cdemkegit